Source/ThirdParty/ANGLE/src/common/log_utils.h

```diff
@@ -248,6 +249,13 @@ std::ostream &FmtHex(std::ostream &os, T value)
 #    define EVENT(message, ...) (void(0))
 #endif
 
+#if defined(ANGLE_ENABLE_ASSERTS)
+static bool IsAssertionEnabled() {
```

Comment:
Having a static function in a header means that function is duplicated in each compilation unit that includes the header.

As suggested already before:

The function could be declared in the header, business as usual. 

The function could be defined in the .cpp file, business as usual.

The function should use the environment query function that ANGLE uses (check the exact function in from the previous review).

The function should compare the string to "1", to avoid the case where the user uses  ´ANGLE_DISABLE_ASSERTS=0`.

The function name should reflect the plurality of the words in related vocabularity:

ANGLE_ENABLE_ASSERTS  -- the preprocessor define is plural

ANGLE_DISABLE_ASSERTS  -- the environment variable  is plural

IsAssertionsEnabled -- the function should be plural